templates/partials/reports_content.html

```diff
@@ -53,7 +53,8 @@
           </li>
           <li ng-if="selection.doc.form && selection.doc.content_type === 'xml'" ng-repeat="field in selection.displayFields" class="indent-{{field.depth}}">
             <label translate>{{field.label}}</label>
-            <p>{{field.value}}</p>
+            <p ng-hide="field.file && field.file.content_type.startsWith('image/')">{{field.value}}</p>
+            <img ng-if="field.file && field.file.content_type.startsWith('image/')" src="data:{{field.file.content_type}};base64,{{field.file.data}}" width="100%"/>
```

Comment:
Don't use `width:100%`. Add a class and add it to the stylesheet, then use `max-width:100%` so small images don't get stretched. Also you'll need `height:auto` so the image will scale equally in both dimensions.

Again if you make the lazy loading change then this would need to change. I'd recommend using an angular directive/filter to load this in. We do a similar thing with the `resourceIcon` filter (via the ResourceIcons service) so you might be able to pull out some reusable component from that.